packages/eth-sdk/src/EthersClient/index.js

```diff
@@ -101,7 +105,29 @@ export default class EthersClient {
     }
 
     const result = await this.explorer.getHistory(address, page, size)
-    return { length: 0, list: result.result }
+    const isHarmony = chainsHarmonyName.includes(this.networkId)
+    const isConflux = chainsConfluxtName.includes(this.networkId)
+    let list = result.result ? (isHarmony ? result.result.transactions : result.result) : (result.list ? result.list : [])
```

Comment:
(result.list ? result.list : [])
it seems like redundant, how about result.list || [] or result?.list ?? []